setup.cfg

```
@@ -37,7 +37,9 @@ packages = find:
 include_package_data = True
 
 install_requires =
-  frozenlist >= 1.1.0
+  frozenlist >= 1.1.0,
+  typing_extensions; python-version == '3.9'
```

(the PyPI project name comes before the semicolon, not after)
```suggestion
  typing-extensions; python_version == '3.9'
```